### ISSUE TITLE
Cant install zend-mail, because zend-mime depends on zend-stdlib: ~2.5

### ISSUE THREAD

**Issue by rommsen** [zendframework/zend-mime#10]
I have installed zend-stdlib 3.0.
Running `composer require zendframework/zend-mail` I get

```
Using version ^2.6 for zendframework/zend-mail
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install zendframework/zend-mail 2.6.1
    - Conclusion: remove zendframework/zend-stdlib 3.0.0
    - Installation request for zendframework/zend-mail ^2.6 -> satisfiable by zendframework/zend-mail[2.6.0, 2.6.1].
    - Conclusion: don't install zendframework/zend-stdlib 3.0.0
    - zendframework/zend-mail 2.6.0 requires zendframework/zend-mime ^2.5 -> satisfiable by zendframework/zend-mime[2.5.0, 2.5.1].
    - zendframework/zend-mime 2.5.0 requires zendframework/zend-stdlib ~2.5.0 -> satisfiable by zendframework/zend-stdlib[2.5.0, 2.5.1, 2.5.2].
    - zendframework/zend-mime 2.5.1 requires zendframework/zend-stdlib ~2.5 -> satisfiable by zendframework/zend-stdlib[2.5.0, 2.5.1, 2.5.2, 2.6.0, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.7.4, 2.7.5, 2.7.6].
    - Can only install one of: zendframework/zend-stdlib[2.5.0, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.5.1, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.5.2, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.6.0, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.7.0, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.7.1, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.7.2, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.7.3, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.7.4, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.7.5, 3.0.0].
    - Can only install one of: zendframework/zend-stdlib[2.7.6, 3.0.0].
    - Installation request for zendframework/zend-stdlib == 3.0.0.0 -> satisfiable by zendframework/zend-stdlib[3.0.0].
```

**Comment by rommsen** [comment]
Sorry for the first report, hit enter accidentally...

**Comment by prolic** [comment]
Why do you want to use Zend\Stdlib 3.0 together with Zend\Mail 2.x?

**Comment by rommsen** [comment]
Because the app I am working on happens to have Zend\Stdlib 3.0 installed. I just want to install Zend\Mail which [has no problem with Zend\Stdlib 3.0](https://github.com/zendframework/zend-mail/blob/master/composer.json#L20) but it also depends on Zend\Mime which [requires ~2.5](https://github.com/zendframework/zend-mime/blob/master/composer.json#L17).

How should I do it?

**Comment by weierophinney** [comment]
For now, `composer require "zendframework/zend-stdlib:^2.7"`. We'll update zend-mime soon to support v3.

**Comment by weierophinney** [comment]
Fixed with #9; release will be cut shortly.
